README.rst

```diff
@@ -40,21 +40,21 @@ Feature extraction:
     .. code-block:: python
 
         >>> from mw.api import Session
-        >>>
+        >>> 
         >>> from revscoring.extractors import APIExtractor
-        >>> from revscoring.features import (bytes_changed, chars_added,
-        ...                                  day_of_week_in_utc,
-        ...                                  hour_of_day_in_utc,
-        ...                                  is_custom_comment,
-        ...                                  user_age_in_seconds, user_is_anon,
-        ...                                  user_is_bot)
+        >>> from revscoring.features import diff, parent_revision, revision, user
         >>>
         >>> api_extractor = APIExtractor(Session("https://en.wikipedia.org/w/api.php"))
         Sending requests with default User-Agent.  Set 'user_agent' on api.Session to quiet this message.
         >>>
-        >>> features = [bytes_changed, chars_added, day_of_week_in_utc,
-        ...               hour_of_day_in_utc, is_custom_comment, user_age_in_seconds,
-        ...               user_is_anon, user_is_bot]
+        >>> features = [revision.day_of_week,
+        ...             revision.hour_of_day,
+        ...             revision.has_custom_comment,
+        ...             parent_revision.bytes_changed,
```

Comment:
Bah.  Now I have changed my mind.  I'm convinced that it will do no harm to move this to diff.  I've just updated the code to do so.